yarn.lock

```diff
@@ -4938,14 +4938,22 @@ lie@~3.3.0:
   dependencies:
     immediate "~3.0.5"
 
-lighthouse-logger@^1.0.0, lighthouse-logger@^1.4.1:
+lighthouse-logger@^1.0.0:
   version "1.4.1"
   resolved "https://registry.yarnpkg.com/lighthouse-logger/-/lighthouse-logger-1.4.1.tgz#a076d7be80cbac16b9fdcd930379b03fff514699"
   integrity sha512-VDZF31xGrLS1Zyf79/Xs6J9I08EVgJyz085MAt0G3Sh6nARFSUaM+IkuEZE9xZrLpTtzH1Qt9UzH/7kt9LnhZQ==
   dependencies:
     debug "^2.6.9"
     marky "^1.2.2"
 
+lighthouse-logger@^2.0.0:
```

Comment:
Looks like LH and chrome-launcher will be using different version of this package. In practice this should be fine since the important global state is stored in marky.

Reply:
So chrome launcher compiles to CJS, which means we would need to make a breaking release to chrome launcher or reintroduce CJS support to the logger.